src/dotnet/ToolPackage/ToolPackageObtainer.cs

```diff
@@ -154,8 +176,15 @@ IProjectRestorer projectRestorer
                     new XElement("PropertyGroup",
                         new XElement("TargetFramework", targetframework),
                         new XElement("RestorePackagesPath", individualToolVersion.Value),
-                        new XElement("RestoreSolutionDirectory", Directory.GetCurrentDirectory()), // https://github.com/NuGet/Home/issues/6199
-                        new XElement("DisableImplicitFrameworkReferences", "true")
+                        new XElement("RestoreProjectStyle", "DotnetToolReference"),
```

Comment:
It was looked over in the design doc you've been usuing as a reference. 
How does the runtimeidentifier get set? 
does the SDK set it?

Reply:
by passing via command line https://github.com/wli3/cli/blob/a1bcedb87b885e52d1fcc862f06ef9fda09ccd99/src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs#L41